tests/Localization/KaGeTest.php

```diff
@@ -97,130 +97,130 @@ class KaGeTest extends LocalizationTestCase
         // Carbon::parse('2018-04-10 00:00:00')->isoFormat('DDDo')
         'მე-100',
         // Carbon::parse('2018-02-10 00:00:00', 'Europe/Paris')->isoFormat('h:mm a z')
-        '12:00 დილის cet',
+        'ღამის 12:00 cet',
```

Comment:
This one will not be possible as the format is explicitly `h:mm a z`, order can be changed in macro-formats (LTS, LL, etc.) but not when the order is given explicitly by `isoFormat`

Reply:
About ღამის, დილის, can you give me the hours range for each?

Reply:
I seen that example in Ru test. Is it mistake as well?

Reply:
ღამის 22:00 - 04:00
დილის 04:00 - 11:00
შუადღის/დღის 11:00 - 16:00
საღამოს 16:00 - 22:00

I'll try to find some official docs about these time ranges